src/request/export/ExportFile.js

```diff
@@ -106,11 +106,13 @@ class ExportFile extends Nimiq.Observable {
      * @param {Key | null} key
      * @param {boolean} isProtected
      */
-    setKey(key, isProtected) {
+    setKey(key, isProtected) { // eslint-disable-line no-unused-vars
```

Comment:
Because if I remove it, I will have to remove it from lots of other places, thereby adding lots of changes to this and other export-related files, creating merge-conflicts for the future Export PR.

Reply:
Might be changed, when the `eligable` will be changed.